mmv1/products/dlp/terraform.yaml

```diff
@@ -114,6 +114,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           trigger: "trigger"
         test_env_vars:
           project: :PROJECT_NAME
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dlp_job_trigger_bigquery_identifying_fields"
+        primary_resource_id: "bigquery_identifying_fields"
+        vars:
+          trigger: "trigger"
```

Comment:
I didn't see this variable being used in the example, do you know if you need it? On a related note, we would prefer to set the trigger's identifier using a variable so that it gets cleaned up in cases where the test fails, see https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-mmv1-test/#terraformyaml-metadata.

Reply:
Done.